src/components/ebay-pagination/index.js

```diff
@@ -88,6 +88,7 @@ function onUpdate() {
 
 function refresh() {
     let current = 0;
+    this.pageContainerEl.style.overflow = 'hidden';
```

Comment:
I think we should rethink this. 
I tried on local and I see what the change here does, but I don't understand why we need the hidden overflow at all. I read the comment but I can't seem to reproduce or see the issue stated there. Can you elaborate?

On the skin side the `.pagination__items` class has `overflow: hidden` but we are setting this here. Should we change that on the skin side?

It seems like the bigger issue is the height of the container is `48px` while the height of the button inside the container is also `48px`. I think we should use flex to grow the button to match the container height but leave enough room for the container.
I'm fine with the overflow fix, but I would say put the `overflow:visible` as the default on skin.

Reply:
We need it in skin because otherwise we'll get this:
![Screen Shot 2019-09-10 at 8 42 32 AM](https://user-images.githubusercontent.com/1562843/64628848-f8942100-d3a6-11e9-9efe-f971f30a402c.png)

I'm not sure how the height of the container would effect this. It's more about the width.

But I'm not thrilled with this solution either, so if anyone can come up with something better, I'm all for it.